addons/account/models/account_move.py

```diff
@@ -2758,6 +2759,30 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
 
         return reverse_moves
 
+    def _action_invoice_ready_to_be_sent(self):
+        """ Hook allowing custom code when an invoice becomes ready to be sent by mail to the customer.
+        For example, when an EDI document must be sent to the government and be signed by it.
+        """
+
```

Comment:
It seems the `pass` has been lost. :dancers:

Reply:
It's ok if there is a comment, the `pass` is only needed for syntax reasons